simde/x86/svml.h

```diff
@@ -3661,6 +3616,91 @@ simde_mm512_mask_exp_pd(simde__m512d src, simde__mmask8 k, simde__m512d a) {
   #define _mm512_mask_exp_pd(src, k, a) simde_mm512_mask_exp_pd(src, k, a)
 #endif
 
+SIMDE_FUNCTION_ATTRIBUTES
+simde__m128
+simde_mm_cdfnorm_ps (simde__m128 a) {
+  #if defined(SIMDE_X86_SVML_NATIVE) && defined(SIMDE_X86_SSE_NATIVE)
+    return _mm_cdfnorm_ps(a);
+  #elif (SIMDE_NATURAL_VECTOR_SIZE > 0)
+
+    simde__m128 a1 = simde_mm_set1_ps(SIMDE_FLOAT32_C( 0.254829592));
+    simde__m128 a2 = simde_mm_set1_ps(SIMDE_FLOAT32_C(-0.284496736));
+    simde__m128 a3 = simde_mm_set1_ps(SIMDE_FLOAT32_C(1.421413741));
+    simde__m128 a4 = simde_mm_set1_ps(SIMDE_FLOAT32_C(-1.453152027));
+    simde__m128 a5 = simde_mm_set1_ps(SIMDE_FLOAT32_C(1.061405429));
+    simde__m128 p = simde_mm_set1_ps(SIMDE_FLOAT32_C(0.3275911));
+    simde__m128 one = simde_mm_set1_ps(SIMDE_FLOAT32_C(1.0));
```

Comment:
These should really all be `static const`.  TBH it probably doesn't matter from the compiler's perspective since it should be able to determine that you never touch the variables anyways, but it also serves as documentation for human readers, and lets the compiler provide you with a bit of protection; if you accidentally try to assign to a `const` variable the compiler will emit an error.  It's just a good habit to get into.